helm_deploy/values-dev.yaml

```diff
@@ -37,7 +37,6 @@ generic-service:
     UNCANCEL_MULTIPLE_SESSIONS_ENABLED: "true"
     NOT_REQUIRED_IN_ADVANCE_ENABLED: "true"
     PRISONER_ALLOCATIONS_ENABLED: "true"
-    BVLS_FEATURE_HMCTS_LINK_GUEST_PIN: "true"
 
```

Comment:
Should we add to our .env files? And does it need to be set to either true or false for Cypress in `feature.env` ?